website/docs/gen-ai/training/bionemo.md

```diff
@@ -0,0 +1,244 @@
+
+#### Clone the repository
+
+First, clone the repository containing the necessary files for deploying the blueprint. Use the following command in your terminal:
+
+```bash
+git clone https://github.com/awslabs/data-on-eks.git
+```
+
+#### Initialize Terraform
+
+Navigate into the directory specific to the blueprint you want to deploy. In this case, we're interested in the BioNeMo blueprint, so navigate to the appropriate directory using the terminal:
+
+```bash
+cd data-on-eks/ai-ml/bionemo
+```
+
+#### Run the install script
+
+Use the provided helper script `install.sh` to run the terraform init and apply commands. By default the script deploys EKS cluster to `us-west-2` region. Update `variables.tf` to change the region. This is also the time to update any other input variables or make any other changes to the terraform template.
+
+
+```bash
+./install .sh
+```
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h3><span>Verify Deployment</span></h3>}>
+
+Update local kubeconfig so we can access kubernetes cluster
+
+```bash
+aws eks update-kubeconfig --name bionemo-on-eks #or whatever you used for EKS cluster name
+```
+
+Since there is no helm chart for Training Operator, we have to manually install the package. If a helm chart gets build by training-operator team, we 
+will incorporate it to the terraform-aws-eks-data-addons repository.
+
+Install Kubeflow Training Operator:
+```bash
+kubectl apply -k "github.com/kubeflow/training-operator/manifests/overlays/standalone?ref=v1.7.0"
+```
+
+First, lets verify that we have worker nodes running in the cluster.
+
+```bash
+kubectl get nodes
+```
+:::info
+```bash
+NAME                                          STATUS   ROLES    AGE   VERSION
+ip-100-64-243-22.us-west-2.compute.internal   Ready    <none>   47h   v1.29.0-eks-5e0fdde
+ip-100-64-70-74.us-west-2.compute.internal    Ready    <none>   47h   v1.29.0-eks-5e0fdde
+...
+```
+:::
+
+Next, lets verify all the pods are running.
+
+```bash
+kubectl get pods -A
+```
+Make sure training-operator, nvidia-device-plugin and fsx-csi-controller pods are running and healthy.
+
+</CollapsibleContent>
+
+
+### Run BioNeMo Training jobs
+
+Once you've ensured that all components are functioning properly, you can proceed to submit jobs to your clusters.
+
+#### Step1: Initiate the Uniref50 Data Preparation Task:
+
+The first task, named the `uniref50-job.yaml`, involves downloading and partitioning the data to enhance processing efficiency. This task specifically retrieves the `uniref50 dataset` and organizes it within the FSx for Lustre Filesystem. This structured layout is designed for training, testing, and validation purposes. You can learn more about the uniref dataset [here](https://www.uniprot.org/help/uniref).
+
+To execute this job, navigate to the `examples\training` directory and deploy the `uniref50-job.yaml` manifest using the following commands:
+
+```bash
+cd training
+kubectl apply -f uniref50-job.yaml
```

Comment:
Show the output of `kubectl get pods` for this job. Explain how many nodes and pods are used by this job.